tests/ipv6_test.go

```diff
@@ -291,15 +339,42 @@ func TestIPv6_AppendRow(t *testing.T) {
 	if !col.Row(3, false).(net.IP).Equal(ip) {
 		require.Failf(t, "Invalid result of AppendRow", "Added %q instead of %q", col.Row(3, false), ip)
 	}
-}
 
-func TestIPv6_Append(t *testing.T) {
-	ips := getTestIPv6()
+	// appending [16]byte
+	require.NoError(t, col.AppendRow(netip.MustParseAddr(ip.String()).As16()))
+	require.Equal(t, 5, col.Rows(), "AppendRow didn't add IP")
```

Comment:
It's better to use `assert.Equal`, since it does not fail immediately, so you see all assertion's effect. Require is good when some prerequisites setup is necessary.